### DISCUSSION TITLE
Is this database allowed?

### DISCUSSION THREAD

**Discussion by ixnoahlive** [HypixelDev/PublicAPI#662] [Q&A]
Hi all. I've recently started work on a massive "database" that polls the `/housing/active` endpoint which to my knowledge is allowed to be polled. It does not send further requests besides this one every 5 minutes* and then logs the house's latest info to a database. I heard someone be blocked for doing something similar with SkyBlock, don't know too much abt that though so I thought I'd ask.

_*and some manual queries which people could make in the future that are under control_

TLDR: I am just keeping a infinite log of every house on the active houses endpoint with their name histories. Is this allowed?

![image](https://github.com/user-attachments/assets/edc94d27-bb21-4436-8547-2638b8854aee)

**Comment by ixnoahlive** [comment]
Anyone?
